public/app/features/explore/ResponseErrorContainer.tsx

```diff
@@ -10,12 +10,7 @@ interface Props {
 export function ResponseErrorContainer(props: Props) {
   const queryResponse = useSelector((state: StoreState) => state.explore[props.exploreId]?.queryResponse);
 
-  // Only show error if it does not have refId. Otherwise let query row to handle it so this condition has to be matched
-  // with QueryRow.tsx so we don't loose errors.
-  const queryError =
-    queryResponse?.state === LoadingState.Error && queryResponse?.error && !queryResponse.error.refId
-      ? queryResponse.error
-      : undefined;
+  const queryError = (queryResponse?.state === LoadingState.Error && queryResponse?.error) || undefined;
```

Comment:
hmm.. nitpick.. could the following be a little simpler?

```suggestion
  const queryError = queryResponse?.state === LoadingState.Error ? queryResponse?.error : undefined;
```